c++/cmake/CapnProtoConfig.cmake.in

```diff
@@ -49,7 +49,10 @@ if(NOT _IMPORT_PREFIX)
   set(_IMPORT_PREFIX ${PACKAGE_PREFIX_DIR})
 endif()
 
-
+if (@WITH_OPENSSL@)
```

Comment:
If CAPNP_LITE is true, then WITH_OPENSSL could still be "AUTO" at this point, I think. Would that make a lite-mode installation default to depend on OpenSSL? (That said, I wonder if there are any lite-mode installations in the wild, even.)